llvm_util/utils.cpp

```diff
@@ -141,6 +141,11 @@ Value* get_operand(llvm::Value *v) {
     return ret;
   }
 
+  if (auto ptr = dyn_cast<llvm::GlobalVariable>(v)) {
```

Comment:
this won't compile on gcc because ptr isn't used and we build with `-Werror` :)

Reply:
```suggestion
  if (isa<llvm::GlobalVariable>(v)) {
```
?